### ISSUE TITLE
cxx can't recognize namespace alias

### ISSUE THREAD

**Issue by idning** [universal-ctags/ctags#992]
namespace a = b;

will cause the whole file unable to get parsed.
```

$ cat a.cpp && ctags a.cpp && cat tags
#include <iostream>
namespace a = b;
int main(){
    std::cout << "helloworld" << endl;
    return 0;
}

ctags: Warning: Language "go" already defined
!_TAG_FILE_FORMAT       2       /extended format; --format=1 will not append ;" to lines/
!_TAG_FILE_SORTED       0       /0=unsorted, 1=sorted, 2=foldcase/
!_TAG_PROGRAM_AUTHOR    Universal Ctags Team    //
!_TAG_PROGRAM_NAME      Universal Ctags /Derived from Exuberant Ctags/
!_TAG_PROGRAM_URL       https://ctags.io/       /official site/
!_TAG_PROGRAM_VERSION   0.0.0   /c11568b/

```

**Comment by idning** [comment]
and the version `8213b2c` works well.

**Comment by masatake** [comment]
Reproduced.

**Comment by pragmaware** [comment]
Should be fixed via #1000 .
